internal/backend/remote-state/pg/backend_test.go

```diff
@@ -205,9 +248,48 @@ func TestBackendConfig(t *testing.T) {
 				t.Fatal("Backend could not be configured")
 			}
 
-			_, err = b.db.Query(fmt.Sprintf("SELECT name, data FROM %s.%s LIMIT 1", schemaName, statesTableName))
-			if err != nil {
-				t.Fatal(err)
+			// Make sure everything has been created
+			if !skipSchemaCreation {
+				// Make sure schema exists
+				var count int
+				query := fmt.Sprintf(`select count(*) from information_schema.schemata where schema_name=$1`)
```

Comment:
from the linter: fmt.Sprintf is unnecessary here and a few other places down below

Reply:
Updated. It should pass golangci-lint now.